src/components/SignalCard/card/SignalCardBottom.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { DesktopOnly } from '../../Responsive'
+import Icon from '@santiment-network/ui/Icon'
+import Toggle from '@santiment-network/ui/Toggle'
+import StatusLabel from '../../StatusLabel'
+import MoreSignalActions from '../controls/MoreSignalActions'
+import styles from './SignalCard.module.scss'
+
+const UnpublishedMsg = () => (
+  <h4 className={styles.awaiting}>
+    <Icon type='clock' className={styles.awaiting__icon} /> Awaiting posting
+  </h4>
+)
+
+const SignalCardBottom = ({
+  signalId,
+  signal,
+  removeSignal,
+  isPublished = true,
+  isAwaiting = false,
+  toggleSignal,
+  isUserTheAuthor = true,
+  deleteEnabled,
+  showMoreActions,
+  showStatus = true,
+  editable = true
+}) => {
+  const { isActive, isPublic, title } = signal
+  return (
+    showStatus && (
+      <div className={styles.bottom}>
+        {showMoreActions && (
+          <DesktopOnly>
+            <MoreSignalActions
+              isUserTheAuthor={isUserTheAuthor}
+              removeSignal={removeSignal}
+              signalTitle={title}
+              signalId={signalId}
+              isPublic={isPublic}
+              deleteEnabled={deleteEnabled}
+              editable={editable}
+            />
+          </DesktopOnly>
+        )}
+
+        {isPublished ? (
+          <h4 className={styles.author}>
+            {isAwaiting && (
+              <div className={styles.awaitingBlock}>
+                <Icon type='awaiting' />
+                <span>&nbsp;&nbsp;Awaiting confirmation</span>
```

Comment:
Do we need this case? As I know - we removed this step when you publish insight

Reply:
It is a signal card. Only moved code in separate file